apps/dashr-studybrowser/app.R

```diff
@@ -0,0 +1,286 @@
+appName <- Sys.getenv("DASH_APP_NAME")
+pathPrefix <- sprintf("/%s/", appName)
+Sys.setenv(DASH_ROUTES_PATHNAME_PREFIX = pathPrefix, 
+           DASH_REQUESTS_PATHNAME_PREFIX = pathPrefix)
+setwd("app/apps/dashr-studybrowser")
+
+library(jsonlite)
+library(dashR)
+library(dashCoreComponents)
+library(dashHtmlComponents)
+library(data.table)
+library(stringr)
+library(magrittr)
+library(foreach)
+library(rlist)
+library(glue)
+library(plotly)
+
+
+
+#Syntactic sugar. just relabeling
+l. <- list
```

Comment:
This is really nice as far as making lists easier to create for you, but since this is example code for R, and R users are likely used to the `list(...)` syntax, it might be better to keep `list(...)`